be/src/exprs/runtime_filter.cpp

```diff
@@ -1174,6 +1174,9 @@ Status IRuntimeFilter::init_with_desc(const TRuntimeFilterDesc* desc, const TQue
     _has_remote_target = desc->has_remote_targets;
     _expr_order = desc->expr_order;
     _opt_remote_rf = desc->__isset.opt_remote_rf && desc->opt_remote_rf;
+    if (desc->__isset.wait_time_ms) {
+        _rf_wait_time_ms = desc->wait_time_ms;
```

Comment:
what value will be assigned to `_rf_wait_time_ms` if `desc->__isset.wait_time_ms` is false?

Reply:
it will be the wait_time in session variable